extensions/twitter-trendscast/package.json

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "twitter-trendscast",
+  "title": "Twitter Trendscast",
+  "description": "With these simple shortcuts, you can keep up with the latest trends from people you follow or any other trends on Twitter.",
+  "icon": "ext-icon.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "twitter-trendscast",
+      "title": "Twitter Trendscast",
```

```suggestion
      "title": "Show Trendscast",
      "subtitle": "Twitter",
```
What do you think about this? I'm not sure if it's better than it is right now

---

I've updated something similar with few changes "Show trends from twitter" @pernielsentikaer